lib/src/Base/Stat/CorrelationAnalysis.cxx

```diff
@@ -129,7 +129,7 @@ Point CorrelationAnalysis::computeSRC(const Sample & sampleX,
                                       const Sample & sampleY) const
 {
   const UnsignedInteger dimension = sampleX.getDimension();
-  if (!(dimension >= 2)) throw InvalidDimensionException(HERE) << "Error: input sample must have dimension > 1, here dimension=" << dimension;
```

Comment:
You mean `sampleY`?

`sampleX` might have dimension >= 2!

Reply:
I don't understand your point. I removed this check after @JPelamatti pointed out it was useless:

https://github.com/openturns/openturns/pull/2056#discussion_r886621668

Reply:
I did not notice the `!` in `!(dimension >= 2)`. Sorry
Everything is ok